README.md

```diff
@@ -0,0 +1,18 @@
+Tenks
+=====
+
+Tenks is a utility that manages virtual bare metal clusters for development and
+testing purposes.
```

Comment:
I'd been led to believe there would be an explanation of the name TENKS here...

Reply:
True, I did say that... I won't lead you on any more of a wild goose chase, the answer is actually in the [Ironic spec](https://review.openstack.org/#/c/579583/5/specs/approved/virtual-bare-metal-cluster-manager.rst).

Reply:
Ah, very pithy.  Nice one @w-miller and @markgoddard